ansible/roles/packages/tasks/url.yaml

```diff
@@ -57,6 +57,7 @@
     mode: 0755
     owner: root
     group: root
+    force: true
```

Comment:
very interesting catch for preprovisioned case. I think we have to scan the KIB code to make sure we fix similar issues with other binaries.

Reply:
Yep I looked at that, this PR was actually a followup to https://d2iq.atlassian.net/browse/D2IQ-95908, we just missed it the first time :(